docs/wiki/Cryptography/post-quantum-cryptography.md

```diff
@@ -0,0 +1,36 @@
+# Post-Quantum Cryptography
+
+Classical cryptography safeguards information by leveraging the inherent difficulty of certain mathematical problems. These problems fall under the area of mathematical research called the ["Hidden Subgroup Problem (HSP)"](https://en.wikipedia.org/wiki/Hidden_subgroup_problem). Imagine a large group with a secret subgroup known only to insiders, these problems makes determining the structure of the secret subgroup (size, elements) computationally intractable for an outsider. Whereas, someone with the "secret" (the private key) can easily identify the subgroup.
+
+Public-key cryptography leverages this concept. Algorithms like RSA, DSA, and [ECDSA](/wiki/Cryptography/ecdsa.md) rely on hidden subgroup problems like prime factorization of large integers or discrete logarithm calculations to secure private keys. The difficulty of solving these problems increases exponentially with key size, making brute-force attacks impractical for classical computers. This inherent difficulty safeguards encrypted data.
```

Comment:
I would make a bit more clear here - ecc relies on discrete logarithm problem and RSA on integer factorization. Especially in context of Ethereum, I would highlight the former. And that both are instances of the hidden subgroup problem, it's implied here but maybe not as clear for a new reader

Reply:
Reworded for more clarity.